experimental/dds/tree/src/SharedTree.ts

```diff
@@ -1505,14 +1505,14 @@ export class SharedTree extends SharedObject<ISharedTreeEvents> implements NodeI
 					default:
 						fail('Unknown version');
 				}
-				this.applyEditLocally(stashedEdit, undefined);
-				return { transformedEdit: stashedEdit };
+				this.applyEditInternal(stashedEdit);
```

Comment:
@microsoft/fluid-cr-tree this looked correct to me, but would be good to have an expert weigh in

Reply:
LGTM.